### ISSUE TITLE
Address the max_record issue raised in csw_search_example.ipynb

### ISSUE THREAD

**Issue by ocefpaf** [ioos/notebooks_demos#65]
@rsignell-usgs comment from `csw_search_example.ipynb`

```
Hmmm..... In the query above, we only get 10 records, even though we specified maxrecords=100.

What's up with that?

Turns out the CSW service specified a MaxRecordDefault that cannot be exceeded. For example, checking: https://dev-catalog.ioos.us/csw?request=GetCapabilities&service=CSW we find:

<ows:Constraint name="MaxRecordDefault">
    <ows:Value>10</ows:Value>
</ows:Constraint>
So we need to loop the getrecords request, incrementing the startposition:
```

We need to loop in every `getrecords` in every notebooks that uses `pycsw`.

**Comment by rsignell-usgs** [comment]
For example, in our boston_light_swim notebook.

**Comment by ocefpaf** [comment]
This is fixed in #137.

We need to replace the calls to `csw.getrecords2` with `get_csw_records(csw, filter_list, pagesize=10, maxrecords=1000)` from the `ioos_tools` module.
